pom.xml

```diff
@@ -3,19 +3,19 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.microsoft.azure</groupId>
-  <artifactId>resources-java-manage-resource-group</artifactId>
+  <artifactId>hybrid-compute-java-manage-vm</artifactId>
```

Comment:
Please revert back the changes regarding name changes. (change "compute" to "resources")